pkg/plugins/resources/memory/store.go

```diff
@@ -74,10 +85,13 @@ func (c *memoryStore) Create(_ context.Context, r model.Resource, fs ...store.Cr
 		return store.ErrorResourceAlreadyExists(r.GetType(), opts.Name, opts.Mesh)
 	}
 
+	now := time.Now()
```

Comment:
Why `creationTime/modificationTime` is set at DAO layer instead of Service layer ?